packages/next/src/server/lib/app-info-log.ts

```diff
@@ -98,6 +99,11 @@ export function logStartInfo({
     }
   }
 
+  // Report available cores to put execution times in perspective
+  // `availableCores` is preferred over `cpus().length` because it reflects any restrictions applied to containers
+  const availableCores = os.availableParallelism()
+  Log.bootstrap(`- Available Cores: ${availableCores}`)
```

Comment:
Maybe CPU architecture and OS platform could be helpful as well for more context.